src/drive/web/modules/drive/Toolbar/components/PortaledQueue.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import ReactDOM from 'react-dom'
-import { UploadQueue } from 'drive/web/modules/upload/UploadQueue'
```

Comment:
I am not sure what's happening here, did it break before ? Does this needs a test ?

Reply:
It works before, but just launching webpack tells us 

```
/src/drive/web/modules/drive/Toolbar/components/PortaledQueue.jsx 39:55-66
"export 'UploadQueue' was not found in 'drive/web/modules/upload/UploadQueue' 
```